src/app/tests/character.service.spec.ts

```diff
@@ -1,12 +1,9 @@
 import { TestBed } from '@angular/core/testing';
-import { HttpClientTestingModule} from '@angular/common/http/testing';
-
+import { HttpClientTestingModule} from "@angular/common/http/testing";
 import { of } from "rxjs";
-
 import Character from '../models/Character';
 import characters from "./data/Character.json";
-import { CharacterService } from '../services/character.service';
-
+import { CharacterService } from "../services/character.service";
```

Comment:
Beware the autocompletes the editor does -- sometimes it uses the wrong quotes. In this case, all the other files use single quotes so we want to be consistent. Just pointing it out, you can change it later.